vm/integer.go

```diff
@@ -282,6 +282,43 @@ var builtinIntegerMethods = []*BuiltInMethod{
 		},
 		Name: "to_s",
 	},
+	{
+		Fn: func(receiver Object) builtinMethodBody {
+			return func(vm *VM, args []Object, blockFrame *callFrame) Object {
+				if len(args) > 0 {
+					return &Error{Message: "Too many arguments for Integer#--"}
```

Comment:
`Integer#even`
This is Ruby's convention, we declare method name using `ClassName#MethodName`. So the below should be `Integer#odd`